### DISCUSSION TITLE
404 NOT FOUND for custom domain

### DISCUSSION THREAD

**Discussion by Success-akinyemi** [vercel/community#3712] [Help] [answered]
### Summary

I am new ti vercel and I have an hobby account. I uploaded my website and custom domain there. But if I visit my website through my domain and login to the dashboard is shows correctly. But if I refresh the dashboard page I get a 404 NOT FOUND page. I believe is not supposed to so. What can I do.

### Steps to Reproduce

It Is a custom domain on vercel hobby account

**Comment by swarnava** [comment]
Hi,

What is your domain name?

**Reply by Success-akinyemi** [reply]
brave.com.ng

**Reply by Success-akinyemi** [reply]
brave.com.ng

**Reply by amyegan** [reply]
Thanks for sharing that. I looked into the DNS records for this domain. It doesn't appear to be configured for use with Vercel.
- https://www.whatsmydns.net/#A/brave.com.ng
- https://www.whatsmydns.net/#NS/brave.com.ng

You can set up a custom domain by following the steps in this guide: https://vercel.com/docs/projects/domains/add-a-domain#

If you have already followed these steps and continue to have trouble, please reach out to your domain provider. They will be able to offer help related to domain settings in their system.

**Reply by Success-akinyemi** [reply, marked as answer]
Thank you all for the reply.
The correct domain name is bravesub.com.ng
The problem actually was Is Is am using Vite + react. So in my project root directory I have to add a vercel.json file and set the rewrites rule there. It is now working well.

**Reply by 504knight** [reply]
@Success-akinyemi I am also using Vite + React and am having this same issue. Can you share what you did in your vercel.json file to setup the rules?

The issue that I am having is that my login and/or home screen will flash for a second and then I get the 404.

**Reply by Success-akinyemi** [reply]
Hi,
The changes I made was. In you root directory create a vercel.json file and add this code:

{
    "rewrites": [
      { "source": "/(.*)", "destination": "/" }
    ]
}
